Pages/forgot.html

```diff
@@ -110,7 +110,7 @@ <h2 class="form-title">Forget Password</h2>
             </form>
         </section>
         <div class="hero-login">
-            <img src="/Assets/Gifs/Reset password.gif password.gif" alt="Reset password">
+            <img src="/Assets/Gifs/Reset password.gif" alt="Reset password">
         </div>
 
```

Comment:
if intentionally you have left space then its fine but in good practice its not doable.
suggestion:remove space